### PR TITLE
Added D403: First word of first line should be properly capitalized

### PR DESCRIPTION
**Pull request by Nurdok**
The new error is turned on by default.

### CODE REVIEW

File: src/pep257.py [hunk 1]
```diff
@@ -128,6 +128,17 @@ def __iter__(self):
     def _publicity(self):
         return {True: 'public', False: 'private'}[self.is_public]
 
+    @property
+    def source(self):
+        """Return the source code for the definition."""
+        full_src = self._source[self._slice]
+
+        def predicate(line):
```
Comment (sigmavirus24) [inline src/pep257.py]: A better name might be nice, e.g., `is_comment_or_empty`. `predicate` is a bit too generic.

File: src/pep257.py [hunk 2]
```diff
@@ -678,6 +689,10 @@ def to_rst(cls):
                          '%r, not %r')
 D402 = D4xx.create_error('D402', 'First line should not be the function\'s '
                                  '"signature"')
+D402 = D4xx.create_error('D402', 'First line should not be the function\'s '
```
Comment (sigmavirus24) [inline src/pep257.py]: Why did you duplicate 402 here?

File: src/pep257.py [hunk 3]
```diff
@@ -1592,6 +1607,18 @@ def check_no_signature(self, function, docstring):  # def context
...
 
+    @check_for(Function)
+    def check_capitalized(self, function, docstring):
+        """D403: First word of the first line should be properly capitalized.
+
+        The [first line of a] docstring is a phrase ending in a period.
+
+        """
+        if docstring:
+            first_word = eval(docstring).split()[0]
```
Comment (sigmavirus24) [inline src/pep257.py]: We should use `ast.literal_eval` instead. I don't think we want people to have docstrings with arbitrary python code to execute in it.

File: src/pep257.py (hunk 1 above)
Comment (Nurdok) [inline src/pep257.py]: Done.

File: src/pep257.py (hunk 2 above)
Comment (Nurdok) [inline src/pep257.py]: By mistake, done.

File: src/pep257.py (hunk 3 above)
Comment (Nurdok) [inline src/pep257.py]: In this case, we use `eval` on a `tk.STRING` token, so it seems like it can't possibly run arbitrary Python code, can it?

Comment (sigmavirus24) [inline src/pep257.py]: That's a good question. I'm not 100% certain. My familiar with the tokenizer library isn't so great.

Comment (myint) [inline src/pep257.py]: @Nurdok, why risk it? Is there some benefit to using `eval()` over the safer `ast.literal_eval()`?

Comment (sigmavirus24) [inline src/pep257.py]: So @Nurdok is correct, what you get back looks like

```py
(3, '"""Biz baz."""', (3, 3), (3, 17), '   """Biz baz."""\n')
```

So you're doing

```py
eval('"""Biz baz."""\n')  # => 'Biz baz.'
```

But it's functionally equivalent to `ast.literal_eval` in this case. I would still prefer `ast.literal_eval` to be used here, but I'm not as stringently against using `eval` as I was earlier.

We should probably add a comment regardless to reassure future reviewers as well.

Comment (Nurdok) [inline src/pep257.py]: I changed it to `ast.literal_eval`, thanks.

**Comment by Nurdok** [comment]
@sigmavirus24 @myint Any more comments before I merge this?

**Comment by myint** [comment]
Looks good to me!

Thanks

**Pull request merged**